README.md

```diff
@@ -39,6 +39,18 @@ is used to check all are present as part of the application start-up. If
 an env var is not found the application will fail to start and an appropriate
 message will be displayed.
 
+Environment variables are used to set application level settings for each
+environment.
+
+| Variable                        | Description                                                               | Default                  | Required |
+|:--------------------------------|:--------------------------------------------------------------------------|:-------------------------|----------|
+| `NODE_ENV`                      | node environment                                                          | development              | | 
+| `PORT`                          | server port                                                               | 3000                     | | 
+| `GOOGLE_ANALYTICS_TRACKING_ID`  | [Google Analytics](https://www.google.co.uk/analytics) property id        |                          | |
+| `WEBTRENDS_TRACKING_ID`         | [Webtrends](https://www.webtrends.com/) tracking id                       |                          | |
```

Comment:
Ha yeah. I was going to mention this in the first place...and the possibility of using the require env vars package but as these aren't required that ins't going to work.